Src/BootCamp.Chapter/Computer/DesktopComputer.cs

```diff
@@ -2,6 +2,15 @@
 {
     public class DesktopComputer
     {
+        public DesktopComputer()
+        {
+            _body = new Body();
```

Comment:
Desktop computer has the same parts both for mac and for windows.
Each factory should be making branded parts and INSTALL those into a new computer.

Reply:
I'm not sure what you mean by "making branded parts" but now I made new parts in each factory and "installed" them in the assemble function.

Reply:
What dos Ms factory have different compared to Win factory? NOthing. Please make sure that Ms factory creates MsRam, MsBody, MsGpu, etc. And Mac creates MacRam, MacCpu, MacBody, etc.

Reply:
Added Ms parts and Mac parts and moved the creation of the parts under the assemble function.